CMakeLists.txt

```diff
@@ -284,7 +286,8 @@ message(STATUS "WITH_SYSTEMD_JOURNAL = ${WITH_SYSTEMD_JOURNAL}")
 message(STATUS "WITH_DOC = ${WITH_DOC}")
 
 message(STATUS "WITH_MAN = ${WITH_MAN}")
-message(STATUS "WITH_DLT_ADAPTOR = ${WITH_DLT_ADAPTOR}")
```

Comment:
@alexmohr 
Please also incorporate comments from @ssugiura at this place as well.
I believe that your PR is ready to go after this minor update.
Thank you

Reply:
re-added `message(STATUS "WITH_DLT_ADAPTOR = ${WITH_DLT_ADAPTOR}")`